lib/game/embedded_game_widget.dart

```diff
@@ -1,15 +1,10 @@
 import 'package:flutter/rendering.dart';
+
 import 'package:flutter/widgets.dart' hide WidgetBuilder;
 
 import 'game.dart';
 import 'game_render_box.dart';
 
-/// This a widget to embed a game inside the Widget tree. You can use it in pair with [BaseGame] or any other more complex [Game], as desired.
-///
-/// It handles for you positioning, size constraints and other factors that arise when your game is embedded within the component tree.
-/// Provided it with a [Game] instance for your game and the optional size of the widget.
-/// Creating this without a fixed size might mess up how other components are rendered with relation to this one in the tree.
-/// You can bind Gesture Recognizers immediately around this to add controls to your widgets, with easy coordinate conversions.
 class EmbeddedGameWidget extends LeafRenderObjectWidget {
```

Comment:
is this class still needed? GameWidget will work on all cases now right?

Reply:
pretty much yes. This is our render box widget, for the overlay stuff we need a stateful one, they have very different concerns.

Reply:
I thought everyone was going to use `GameWidget(game: game)` now instead of `EmbeddedGameWidget(game.widget)`. This was used to add the game as a widget inside a regular app. The current `GameWidget` already provide that right?

Reply:
EmbeddedGameWidget is internal and should not be used outside the package.

Reply:
Removed it in favor of a private class `_GameRenderObjectWidget`.